main.py

```diff
@@ -0,0 +1,11 @@
+from flask import render_template, request
+
+import models.survey
+
+from app import app, db
+
```

Comment:
Maybe all of the routing should be elsewhere? Something like this:

<img width="561" alt="main controllers" src="https://user-images.githubusercontent.com/20856645/37245467-f4d3a27c-24a0-11e8-8665-07f038b6561d.png">

And main could then only contain someting along these lines:
```
if __name__ == '__main__':
        app.debug = True
        app.run(host = '0.0.0.0', port=5000)
```

Reply:
For me all this is OK - it was my first Flask-attempt based on 2-3 tutorials :D. There must be a lot of better ways to do some of the stuff - if so, let me know and I'll change it 8)

Reply:
I tried to add ../main/controllers.py with only code from above picture, and replaced main.py-code with only:

if __name__ == '__main__':
        app.debug = True
        app.run(host = '0.0.0.0', port=5000)

I didnt get it working... could you maybe try to make the change or tell what Im doing wrong :)?